examples/gallery/symbols/velo_arrow_ellipse.py

```diff
@@ -0,0 +1,42 @@
+"""
```

Comment:
Just one last-minute comment: 

This gallery example should be put in the "Seismology and Geodesy" categroy.

Reply:
You're lucky, I was just about to merge this in. Looking closely, I'm actually more tempted to put this under "Lines and vectors" as there doesn't seem to be anything seismology or geodetic about `velo` (though I'm not experienced in either of those fields).

Reply:
The "velo" module is in the "geodesy" supplement (https://docs.generic-mapping-tools.org/dev/supplements/geodesy/velo.html), and the input columns like "eastward, northward velocity, and uncertainty of eastward, northward velocities" are also specific to geodesy applications.

Reply:
Fair enough, had a closer read at the [Geodesy wikipedia page](https://en.wikipedia.org/wiki/Geodesy) page and it seems to fit. I'll move the gallery example.